stories/data.js

```diff
@@ -181,4 +181,9 @@ const largeDataset = [
   }
 ]
 
-export { largeDataset }
+export const monsters = [
```

Comment:
should we leave it in the story file? so they can see what the data looks like from storybook

Reply:
needed this for test. also it's readily available in storybook view

![default](https://user-images.githubusercontent.com/2837532/53910160-317ec200-4021-11e9-8131-2f1a9e4ff3ba.png)